apps/server/src/services/logger/local-logger.ts

```diff
@@ -2,12 +2,17 @@ import { addColors, createLogger, format, Logger, transports } from 'winston';
 
 import { logLevels } from './levels';
 
-const { printf, timestamp, combine, colorize, errors, label } = format;
+const { printf, timestamp, combine, colorize, errors } = format;
 
 const logFormat = printf(
-  ({ subject, label, level, message, timestamp, ...args }) => {
-    return `${label} (${subject}) ${level}: ${timestamp} - ${message} - ${JSON.stringify(
-      args,
+  ({ service, level, message, timestamp, context, ...args }) => {
+    const { subject, ...contextData } = context;
+    const additionalData = {
+      contextData,
```

Comment:
why are we not directly using `context`
?

Reply:
To get the subject out of the context and display it in front of the log 

![image](https://user-images.githubusercontent.com/33794760/229705709-571bf469-5ad7-4675-8aec-1b98bf7bc4ea.png)